imap_processing/packet_definitions/codice/lo_sw_species_counts.xml

```diff
@@ -133,24 +133,45 @@
 						<xtce:Comparison parameterRef="PKT_APID" value="1156" useCalibratedValue="false" />
 					</xtce:RestrictionCriteria>
 				</xtce:BaseContainer>
+				<xtce:EntryList />
+			</xtce:SequenceContainer>
+		</xtce:ContainerSet>
+		<xtce:ContainerSet>
+			<xtce:SequenceContainer name="CCSDSPacket">
 				<xtce:EntryList>
-					<xtce:ParameterRefEntry parameterRef="Spin_Period" />
```

Comment:
It looks like this did change the XTCE output - is this still what you want the XTCE to look like?